modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -0,0 +1,88 @@
++
+----
+$ oc --insecure-skip-tls-verify adm node-logs --role=master --path=openshift-apiserver/
+
+ip-10-0-140-97.ec2.internal audit-2019-04-09T00-12-19.834.log
+ip-10-0-140-97.ec2.internal audit-2019-04-09T11-13-00.469.log
+ip-10-0-140-97.ec2.internal audit.log
+ip-10-0-153-35.ec2.internal audit-2019-04-09T00-11-49.835.log
+ip-10-0-153-35.ec2.internal audit-2019-04-09T11-08-30.469.log
+ip-10-0-153-35.ec2.internal audit.log
+ip-10-0-170-165.ec2.internal audit-2019-04-09T00-13-00.128.log
+ip-10-0-170-165.ec2.internal audit-2019-04-09T11-10-04.082.log
+ip-10-0-170-165.ec2.internal audit.log
+----
+
+.. View the {product-title} API server log for a specific master node and timestamp or view all the logs for that master:
++
+----
+$ oc adm node-logs <log-name> --path=openshift-apiserver/audit.log
+----
++
+For example:
++
+----
+$ oc adm node-logs ip-10-0-140-97.ec2.internal audit-2019-04-08T13-09-01.227.log --path=openshift-apiserver/audit.log
+$ oc adm node-logs ip-10-0-140-97.ec2.internal audit.log --path=openshift-apiserver/audit.log
+----
++
+The output appears similar to the following:
++
+----
+ip-10-0-140-97.ec2.internal {"kind":"Event","apiVersion":"audit.k8s.io/v1beta1","metadata":{"creationTimestamp":"2019-04-09T18:52:03Z"},"level":"Metadata","timestamp":"2019-04-09T18:52:03Z","auditID":"9708b50d-8956-4c87-b9eb-a53ba054c13d","stage":"ResponseComplete","requestURI":"/","verb":"get","user":{"username":"system:anonymous","groups":["system:unauthenticated"]},"sourceIPs":["10.128.0.1"],"userAgent":"Go-http-client/2.0","responseStatus":{"metadata":{},"code":200},"requestReceivedTimestamp":"2019-04-09T18:52:03.914638Z","stageTimestamp":"2019-04-09T18:52:03.915080Z","annotations":{"authorization.k8s.io/decision":"allow","authorization.k8s.io/reason":"RBAC: allowed by ClusterRoleBinding \"cluster-status-binding\" of ClusterRole \"cluster-status\" to Group \"system:unauthenticated\""}}
+----
+
+. View the Kubernetes API server logs:
+
+.. If necessary, get the name of the log you want to view:
++
+----
+$ oc --insecure-skip-tls-verify adm node-logs --role=master --path=kube-apiserver/
+
+ip-10-0-140-97.ec2.internal audit-2019-04-09T14-07-27.129.log
+ip-10-0-140-97.ec2.internal audit-2019-04-09T19-18-32.542.log
+ip-10-0-140-97.ec2.internal audit.log
+ip-10-0-153-35.ec2.internal audit-2019-04-09T19-24-22.620.log
+ip-10-0-153-35.ec2.internal audit-2019-04-09T19-51-30.905.log
+ip-10-0-153-35.ec2.internal audit.log
+ip-10-0-170-165.ec2.internal audit-2019-04-09T18-37-07.511.log
+ip-10-0-170-165.ec2.internal audit-2019-04-09T19-21-14.371.log
+ip-10-0-170-165.ec2.internal audit.log
+----
+
+.. View the Kubernetes API server log for a specific master node and timestamp or view all the logs for that master:
++
+----
+$ oc adm node-logs <log-name> --path=kube-apiserver/audit.log
+----
++
+For example:
++
+----
+$ oc adm node-logs ip-10-0-140-97.ec2.internal audit-2019-04-09T14-07-27.129.log --path=kube-apiserver/audit.log
```

Comment:
Should be `oc adm node-logs ip-10-0-140-97.ec2.internal audit-2019-04-09T14-07-27.129.log --path=kube-apiserver/audit-2019-04-09T14-07-27.129.log`